tests/TestCase/Error/ExceptionRendererTest.php

```diff
@@ -761,6 +761,9 @@ public function testMissingLayoutPathRenderSafe()
         $ExceptionRenderer->render();
         $this->assertEquals('', $ExceptionRenderer->controller->viewBuilder()->layoutPath());
         $this->assertEquals('Error', $ExceptionRenderer->controller->viewBuilder()->templatePath());
+
+        // Just to ensure the callback has actually been triggered, so we're actually testing something:
+        $this->assertTrue($eventTriggered);
```

Comment:
We do this in other places as well. Usually we use instance properties on the tests instead of values passed by reference though